src/test/java/api/requests/scenarios/SingleOpenHoldShelfRequestTests.java

```diff
@@ -113,7 +113,7 @@ public void itemCannotBeCheckedOutToOtherPatronWhenRequestIsAwaitingPickup()
     assertThat(response.getJson(), hasErrorWith(allOf(
       hasMessage("The Long Way to a Small, Angry Planet (Barcode: 036000291452) " +
         "cannot be checked out to user Stuart, Rebecca " +
-        "because it is awaiting pickup by another patron"),
+        "because it has been requested by another patron"),
```

Comment:
Do we have anyway to reference this message as a constant, since this is the third place I have seen it used? This may not be desirable, since it would require that all use cases in tests use the same user and item.

Reply:
This is probably a bias of mine. I prefer literals in tests where possible, I think they aid comprehension (vs. something like `hasMessage(expectedRequestedByOtherPatronMessage(item))`

A method like that could be easily extracted if we think that could be more maintainable / readable, what do you think?

Reply:
I think it would be more maintainable at the cost of readability. I'll defer to you which is preferable.

Reply:
Have extracted a method. 

(Personally, I find it too similar to the implementation, that it loses credibility and maintainability)